pkg/controller/controller.go

```diff
@@ -446,6 +448,9 @@ func (c *Controller) handleClusterUpdate(cluster *myspec.M3DBCluster) error {
 		return err
 	}
 
+	// check if any pods inside the cluster need to be swapped in
+	c.checkPodsForReplacement(cluster, pods, placement)
```

Comment:
Do we need to call `replacePodInPlacement` on the result we get here?

Reply:
yes oops my bad lol